GOVERNANCE.md

```diff
@@ -4,83 +4,139 @@ This document defines project governance for the Strimzi project.
 
-The component owners are responsible for development of a specific subproject or component within Strimzi.
-Such components might be represented by a separate GitHub repository within the Strimzi organization (e.g. `strimzi-kafka-bridge`) or by a subdirectory in a GitHub repository (e.g. `./documentation/` in `strimzi-kafka-operator`).
-The components owners are identified in the [`COMPONENT-OWNERS`](COMPONENT-OWNERS) file in this repository which also includes the component for which they are responsible.
+New maintainers are proposed by an existing maintainer and are elected by a vote (see later for the rules of this vote).
+If the vote succeeds the proposed maintainer will be invited to become a maintainer.
+If the proposed maintainer accepts the invitation:
+* Their name will be added to the `MAINTAINERS` file.
+* They will be added to the collaborators list of the Strimzi repository with "Write" access
+* After 6 months they will be given "Admin" access to the Strimzi repository.
+
+Maintainers can be removed by a vote.
+If the vote succeeds:
+* Their name will be removed from the `MAINTAINERS` file.
+* They will be removed to the collaborators list of the Strimzi repository
 
-By definition, every maintainer is also an owner for all components and does not have to be mentioned in the `OWNERS` list.
+Maintainers who wish to become emeritus simply mark the fact by opening a PR against this repository moving their name to the `EMERITUS` file.
+The process for reverting from an emeritus maintainer to an active maintainer is the same as for new maintainers.
 
-#### Changes in component ownership
+### Changes in component ownership
 
-New component owners can be proposed by any maintainer and are elected by a ⅔ majority maintainers vote.
-Component owners can be removed by a ⅔ majority maintainers vote.
+New component owners can be proposed by any maintainer and are elected by a vote.
+If the vote succeeds the proposed component owner will be invited to become a component owner.
+If the proposed component owner accepts the invitation:
+* Their name will be added to the `COMPONENT-OWNERS` file.
+* Owners of components which have their own GitHub repository will get "Write" rights for given GitHub repository and will be able to merge approved PRs.
+* Owners will not get "Admin" rights on any Strimzi GitHub repositories.
 
-#### Github Project Administration
+Component owners can be removed by a vote.
+If the vote succeeds:
+* Their name will be removed from the `COMPONENT-OWNERS` file.
+* They will be removed to the collaborators list of the Strimzi repository, if any.
 
-Owners of components which have their own GitHub repository will get "Write" rights for given GitHub repository and will be able to merge approved PRs.
-Owners will not get "Admin" rights on any Strimzi GitHub repositories.
+Component owners who wish to become emeritus simply mark the fact by opening a PR against this repository moving their name to the `EMERITUS` file.
```

Comment:
What happens if they don't do this, but don't really contribute any more?

Reply:
See my response to Paolo's [question, above](https://github.com/strimzi/governance/pull/21/files#r1740154232).